processor/spanmetricsprocessor/processor.go

```diff
@@ -326,7 +328,7 @@ func (p *processorImp) aggregateMetricsForSpan(serviceName string, span pdata.Sp
 	p.cache(serviceName, span, key, resourceAttr)
 	p.updateCallMetrics(key)
 	p.updateLatencyMetrics(key, latencyInMilliseconds, index)
```

Comment:
Would the same problem exist for `updateLatencyMetrics` here?

Reply:
From what I understand, the `updateLatencyMetrics  ` seems to be ok.